src/main/java/ava/exception/NoDescriptionException.java

```diff
@@ -0,0 +1,15 @@
+package ava.exception;
+
+/**
+ * Class to represent Duke.Exception.NoDescriptionException.
```

Comment:
Ditto. Is this a typo with the documentation?

```suggestion
 * Class to represent Ava.Exception.NoDescriptionException.
```